Sources/Bag.swift

```diff
@@ -13,14 +13,14 @@ public struct Bag<Element> {
 	public struct Token {
 		fileprivate let value: UInt64
 	}
-
+	
```

Comment:
Could you please remove those white space changes?

Reply:
done